lib/endpoints/class-wp-rest-controller.php

```diff
@@ -127,10 +127,12 @@ public function delete_item( $request ) {
 	/**
 	 * Prepare the item for create or update operation.
 	 *
-	 * @param WP_REST_Request $request Request object.
-	 * @return WP_Error|object $prepared_item
+	 * @param WP_REST_Request $request  Request object.
+	 * @param bool            $creating Whether the item is being created.
```

Comment:
> As for the terms, maybe there's room for an adjustment there as well? I'm seeing this line twice:

I believe that's intentional:

1. Fetch the item from the database.
2. Fetch the item again after it's been updated in the database.